content/guides/references/experiments.md

```diff
@@ -39,11 +39,50 @@ creating `e2e` and `component` objects inside your Cypress configuration.
 These experiments are available to be specified inside the `e2e` configuration
 object:
 
-| Option                           | Default | Description                                                                                               |
-| -------------------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
-| `experimentalStudio`             | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would. |
-| `experimentalRunAllSpecs`        | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.            |
-| `experimentalOriginDependencies` | `false` | Enables support for `require`/`import` within `cy.origin`.                                                |
+| Option                            | Default | Description                                                                                               |
+| --------------------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `experimentalStudio`              | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would. |
+| `experimentalRunAllSpecs`         | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.            |
+| `experimentalOriginDependencies`  | `false` | Enables support for `require`/`import` within `cy.origin`.                                                |
+| `experimentalSkipDomainInjection` | `null`  | Disables injecting `document.domain` into `text/html` pages.                                              |
+
+#### Experimental Skip Domain Injection
+
+Under the hood, Cypress
+[injects document.domain](/guides/guides/web-security#Examples-of-what-Cypress-does-under-the-hood)
+into your test application. In some known cases, this causes issues with certain
+sites. For example, Salesforce executes code in isolated window contexts. When
+Cypress injects `document.domain`, these window contexts become cross-origin and
+fail to communicate. To work around this, the `experimentalSkipDomainInjection`
+flag was introduced.
+
+This flag disables injecting `document.domain` inside Cypress, allowing Cypress
+to test these exception cases. Only enable this flag if you are testing against
+Salesforce or suspect you may have `document.domain` issues. When enabled, this
```

Comment:
maybe put this as a note at the bottom. do we know examples of what might be a document.domain issue?

Reply:
Should we also say Salesforce, Google or ...?

Reply:
Do you mean mention them by name or have something more generic?

I don't think we have easily digestible examples of what might be a document.domain issue. Usually the site just doesn't work inside Cypress, which is _might_ be a document.domain problem

Reply:
yeah we don't have clear examples to share. maybe removing the wording to indicate exception cases, it'll sound less like these are cases the user should know about?

```md
This `experimentalSkipDomainInjection` option disables injecting `document.domain` inside Cypress. When enabled, all cross-origin/subdomain navigation must user `cy.origin()`. This can make tests a bit more verbose and we only recommending including your site pattern if you are having issues running Cypress out of the box and suspect setting `document.domain` is interfering with your site's ability to render properly.

Before enabling, verify your application is not implementing frame busting techniques. **link**. 

At this point in time, we are aware of the following sites that require this to be set to be tested properly:
- Google
- Salesforce
```

Reply:
I gave this a shot in dce0200a8. Let me know what you think!

Reply:
like it!